src/protocol.py

```diff
@@ -210,7 +292,7 @@ def checkIPv6Address(host, hostStandardFormat, private=False):
         if not private:
             logger.debug('Ignoring local address: %s', hostStandardFormat)
         return hostStandardFormat if private else False
-    if (ord(host[0]) & 0xfe) == 0xfc:
+    if (ord(str(host[0])) & 0xfe) == 0xfc:
```

Comment:
I don't know if this is ok as there are no tests for this. So you need to write a test.

Reply:
Okay, I'll write one

Reply:
I've already solved this in 7e1f6e8. Yes, at first I wrote more tests.

Reply:
Okay, I'll merge your changes